docs/mctpd.md

```diff
@@ -1,6 +1,6 @@
 # `mctpd`
 
-## D-Bus
+## D-Bus /au/com/codeconstruct/mctp1/<NetworkId>/endpoints/<EID>
```

Comment:
This section describes both the root object `.../mctp1` as well as the endpoint objects `.../mctp1/networks/<net>/endpoints/<eid>`, so I don't think we need change this in the section title.

(arguably, we should split that up, but that's a separate change...)